BodyText/tests/BodyText-specs.js

```diff
@@ -1,39 +1,51 @@
-import {mount} from 'enzyme';
+import '@testing-library/jest-dom';
+import {render} from '@testing-library/react';
+
 import BodyText from '../BodyText';
-import css from '../BodyText.module.less';
 
 describe('BodyText Specs', () => {
 
-	test('should support multi-line content', () => {
-		const subject = mount(
-			<BodyText />
-		);
-
-		const expected = 1;
-		const actual = subject.find('p').length;
+	test('should render a text', () => {
+		const {getByTestId} = render(<BodyText data-testid="bodyText" />);
+		const bodyText = getByTestId('bodyText');
 
-		expect(actual).toBe(expected);
+		expect(bodyText).toBeInTheDocument();
 	});
 
 	test('should support single-line marqueeing content when `noWrap` is true', () => {
-		const subject = mount(
-			<BodyText noWrap />
-		);
+		const {getByTestId} = render(<BodyText data-testid="bodyText" noWrap />);
+		const bodyText = getByTestId('bodyText');
+		const marquee = bodyText.children.item(0);
 
-		const expected = true;
-		const actual = subject.findWhere(c => c.name() === 'ui:Marquee').exists();
+		const expected = 'marquee';
 
-		expect(actual).toBe(expected);
+		expect(marquee.className).toBe(expected);
```

Comment:
Can you place marquee.className in an const actual, we should keep the actual => expected notation
(same for the ones below)